src/js/components/settings/settings.js

```diff
@@ -50,12 +50,12 @@ export const Settings = ({ allowUserManagement, currentUser, hasMultitenancy, hi
     'user-management': { admin: false, enterprise: false, multitenancy: false, userManagement: true, component: <UserManagement />, text: 'User management' },
     'role-management': { admin: true, enterprise: true, multitenancy: false, userManagement: false, component: <Roles />, text: 'Roles' },
     upgrade: {
-      admin: false,
-      enterprise: false,
+      admin: true,
+      enterprise: true,
       multitenancy: true,
       trial: true,
-      userManagement: false,
-      component: <Upgrade history={history} />,
+      userManagement: true,
```

Comment:
I could see this only being allowed for admins (though I personally would accept an upgrade from every user 😅) but `enterprise` and `userManagement` are probably not required right?